### PR TITLE
Fix the alignments in the All Pages breaking from the navbar

### LINKED ISSUE #1310: Fix the alignments in the All Pages breaking from the navbar
**Describe the bug**
The alignments in the body of the All Pages are ugly. The problem is in the navbar.

**To Reproduce**
1. Go to 'All Pages'

**Expected behavior**
Fix the alignments in the Navbar

**Screenshots**
Engineering Team Page
image

Upspeak
image

IRC
image

ScholarX
image

Student Ambassador
image

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #1310 

## Goals

Fixed the alignments in the All Pages breaking from the navbar

## Approach

Fixing the alignments in the All Pages breaking from the navbar and formatted the code

### Screenshots

One screenshot of one fixed page, there are lot of pages
image

  
### Preview Link

https://pr-1342-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by YoshithaRathnayake** [comment]
@Piumal1999 can you review this, there will be more changes bcs I formatted the code also

**Comment by anjula-sack** [comment]
Can you fix the conflicts? @YoshithaRathnayake

File: irc.html [hunk 1]
```diff
@@ -30,338 +32,319 @@
...
-    <div id="navbar">
-
-    </div>
-</header>
+    <main>
+        <header class="header-global">
+            <!--nav bar goes here-->
+            <div id="navbar"></div>
+        </header>
+    </main>
```
Comment (Piumal1999) [inline irc.html:40]: @YoshithaRathnayake  Normally we use the `main` tag to wrap the non-repeating unique set of elements in a page. So it is not a good practice to put the navbar inside the main tag. 

Btw this alignment issue is related to some kind of overflow property of the navbar. Try using `overflow-hidden` class with the navbar.

Comment (YoshithaRathnayake) [reply irc.html:40]: Ok @Piumal1999

Comment (YoshithaRathnayake) [reply irc.html:40]: > @YoshithaRathnayake Normally we use the `main` tag to wrap the non-repeating unique set of elements in a page. So it is not a good practice to put the navbar inside the main tag.
> 
> Btw this alignment issue is related to some kind of overflow property of the navbar. Try using `overflow-hidden` class with the navbar.

@Piumal1999 I changed it but can't reproduce the bug from overflow hidden

Comment (YoshithaRathnayake) [reply irc.html:40]: > > @YoshithaRathnayake Normally we use the `main` tag to wrap the non-repeating unique set of elements in a page. So it is not a good practice to put the navbar inside the main tag.
> > Btw this alignment issue is related to some kind of overflow property of the navbar. Try using `overflow-hidden` class with the navbar.
> 
> @Piumal1999 I changed it but can't reproduce the bug from overflow hidden

No sorry @Piumal1999 the bug is fixing but when I put overflow hidden the nav dropdowns are hiding

Comment (YoshithaRathnayake) [reply irc.html:40]: Resolved @Piumal1999 the problem was the width of the join us dropdown so decreased the width by dropdown-menu-lg

**Pull request closed without merging**